fili-core/src/main/java/com/yahoo/bard/webservice/web/util/PaginationParameters.java

```diff
@@ -90,6 +93,11 @@ private int parseParameter (String parameter, String parameterName) throws BadPa
             throw new BadPaginationException(errorMessage.format(parameterName));
         }
         try {
+            if (parameter.equals(FIRST)) {
+                return 1;
+            } else if (parameter.equals(LAST)) {
+                return LAST_PAGE;
```

Comment:
Ok if we introduce python-style paging I would definitely be ok with this approach. I don't want to start using negative values as special flags though